code/modules/vore/eating/leave_remains_vr.dm

```diff
@@ -1,75 +1,65 @@
-/obj/belly/proc/handle_remains_leaving(var/mob/living/M)
+/datum/species
+	var/skull_type = /obj/item/weapon/digestion_remains/skull
+/datum/species/tajaran
+	skull_type = /obj/item/weapon/digestion_remains/skull/tajaran
+/datum/species/unathi
+	skull_type = /obj/item/weapon/digestion_remains/skull/unathi
+/datum/species/skrell
+	skull_type = /obj/item/weapon/digestion_remains/skull/skrell
+/datum/species/spider
+	skull_type = /obj/item/weapon/digestion_remains/skull/vasilissan
+/datum/species/akula
+	skull_type = /obj/item/weapon/digestion_remains/skull/akula
+/datum/species/harpy
+	skull_type = /obj/item/weapon/digestion_remains/skull/rapala
+/datum/species/vulpkanin
+	skull_type = /obj/item/weapon/digestion_remains/skull/vulpkanin
+/datum/species/sergal
+	skull_type = /obj/item/weapon/digestion_remains/skull/sergal
+/datum/species/fl_zorren
+	skull_type = /obj/item/weapon/digestion_remains/skull/zorren
+/datum/species/hi_zorren
+	skull_type = /obj/item/weapon/digestion_remains/skull/zorren
+/datum/species/nevrean
+	skull_type = /obj/item/weapon/digestion_remains/skull/nevrean
+/datum/species/teshari
+	skull_type = /obj/item/weapon/digestion_remains/skull/teshari
+/datum/species/vox
+	skull_type = /obj/item/weapon/digestion_remains/skull/vox
+/datum/species/xenohybrid
+	skull_type = /obj/item/weapon/digestion_remains/skull/xenohybrid
 
-	if(istype(M,/mob/living/carbon/human))	//Are we even humanoid?
+/obj/belly/proc/handle_remains_leaving(var/mob/living/M)
+	if(ishuman(M))	//Are we even humanoid?
 		var/mob/living/carbon/human/H = M
```

Comment:
I'd prefer `if(!ishuman(M)) return` just because you can unindent the entire proc by one after it.

Reply:
Agreed